### PR TITLE
add sampling rate to component

### PR DESCRIPTION
**Pull request by ozym**
Turns out there's an edge case which means that some responses fall in the middle where they don't need the response of the datalogger or sensor, but do need a derived response. The part that indicates this is that a sampling rate is needed for the sensor response to allow it to be matched to a stream.

This is mainly used for things like gain changes (i.e. height conversions etc.).

### CODE REVIEW

File: meta/component.go [hunk 1]
```diff
@@ -130,20 +137,34 @@ func (s *ComponentList) decode(data [][]string) error {
 			azimuth = v
 		}
 
+		var samplingRate float64
+		if s := strings.TrimSpace(d[componentSamplingRate]); s != "" {
+			v, err := strconv.ParseFloat(d[componentSamplingRate], 64)
```
Comment (junghao) [inline meta/component.go:142]: Maybe `strconv.ParseFloat(s, 64)`?

**Review by junghao — CHANGES_REQUESTED** [review]
A small change.

File: meta/component.go (hunk 1 above)
Comment (ozym) [reply meta/component.go]: thanks @junghao , good catch.

**Review by junghao — APPROVED** [review]
Looks good

**Pull request merged**